styles/index.css

```diff
@@ -15,21 +15,118 @@ body {
 
```

Comment:
Above you specify a width and height for the body. Please avoid that to make it responsive. You can use the browser's developer tools to get a mobile view for development.